packages/app/src/server/server.ts

```diff
@@ -84,6 +85,15 @@ function createServer() {
   }
   app.use(sessionParser);
 
+  app.use(function sessionCheck(req, res, next) {
+    if (!req.session) {
+      return next(
+        new GeneralServerError(500, "connection to session backend lost")
```

Comment:
About the topic of potentially misleading error messages: are we sure that "connection to session backend lost" is correct / useful in all cases when this condition (`!req.session`) is true? As in: TCP connection lost? In which cases is `!req.session` true, architecturally? Is there any original error message here exposed by the framework/library that we could emit instead / additionally?

Reply:
This is recommended by the library itself: https://github.com/tj/connect-redis#how-do-i-handle-lost-connections-to-redis

Reply:
Of course, in practice, we shall see

Reply:
To be fair - they write `return next(new Error('oh no'))` which is _sad_. Error detail being swallowed.